docs/citing.rst

```diff
@@ -222,6 +222,30 @@ we recommend to include in your paper.
    </p>
 
 
+Note for reviewers and editors
+------------------------------
+
+The boilerplate text generated by fMRIPrep is intended to allow for clear,
+consistent description of the preprocessing steps used, in order to improve the
+reproducibility of studies.
+We fully intend for it to be copied verbatim, and have released it under the
+`CC0 license <https://creativecommons.org/publicdomain/zero/1.0/>`_, dedicating
```

Comment:
I believe you meant this:

```suggestion
`CC0 license <https://creativecommons.org/publicdomain/zero/1.0/>`__, dedicating
```

although it does not make any big difference. Feel free to discard the edit. Happy to merge anytime you say it is finished!.

Reply:
What does a second underscore do?

Reply:
Turns the link into "anonymous" so the title (`CCO license` in this case) can be used at other points of the documentation. If you add the same code with only one underscore somewhere else, sphinx will throw a warning (and fail in our case).

As I said, the likelyhood of having another instance of 
```
`CC0 license <https://creativecommons.org/publicdomain/zero/1.0/>`_
```
is minimal, so we can merge right away without further thoughts.